### PR TITLE
Fix tests broken due the OIDC RP-Initiated Logout

### LINKED ISSUE #311: Some tests broken due the OIDC RP-Initiated Logout
### Describe the bug

It seems there are few test failures, which are probably caused by OIDC RP-Initiated Logout  https://github.com/keycloak/keycloak/issues/10885 . Some details:

```
Tests in error: 
  ArquillianAuthzUMATest.testRequestEntitlements:181 » NoSuchElement no such ele...
  ArquillianAuthzUMATest.testShareResource:200 » NoSuchElement no such element: ...
....
Tests in error: 
  ArquillianAuthzUMATest.testShareResource:170 » NoSuchElement no such element: ...
....

Results :

Tests in error: 
  ArquillianJeeHtml5Test.testAdminWithAuthAndRole:145 » NoSuchElement no such el...

Tests run: 5, Failures: 0, Errors: 1, Skipped: 0
.....

 FAILED TESTS:
- app-authz-photoz
- app-authz-uma-photoz
- app-jee-html5
Check individual logs for details
```

### Version

18.0.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by mposolda**
Closes #311

### CODE REVIEW

**Comment by mposolda** [comment]
@pedroigor @stianst Anyone of you has a chance to review this?

File: app-authz-uma-photoz/photoz-testsuite/src/test/java/org/keycloak/quickstart/uma/page/LogoutConfirmPage.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    public void confirmLogout() {
+        waitGui().until().element(confirmLogoutButton).is().clickable();
+        confirmLogoutButton.click();
+    }
+
+    public boolean isUserLoggedOut() {
+        waitGui().until().element(infoMessage).is().present();
+        return "You are logged out".equals(infoMessage.getText());
+    }
```
Comment (vmuzikar) [inline app-authz-uma-photoz/photoz-testsuite/src/test/java/org/keycloak/quickstart/uma/page/LogoutConfirmPage.java:27]: Just a nitpick. I don't think this is used anywhere.

**Review by stianst — DISMISSED** [review]

**Review by vmuzikar — APPROVED** [review]

**Review by pedroigor — APPROVED** [review]

File: app-authz-uma-photoz/photoz-testsuite/src/test/java/org/keycloak/quickstart/uma/page/LogoutConfirmPage.java (hunk 1 above)
Comment (mposolda) [reply app-authz-uma-photoz/photoz-testsuite/src/test/java/org/keycloak/quickstart/uma/page/LogoutConfirmPage.java]: Yes, true. I removed this.

Comment (mposolda) [reply app-authz-uma-photoz/photoz-testsuite/src/test/java/org/keycloak/quickstart/uma/page/LogoutConfirmPage.java]: Good catch BTV :)

**Review by vmuzikar — APPROVED** [review]

**Comment by mposolda** [comment]
@vmuzikar @stianst @pedroigor Thanks you all for the review!

**Pull request merged**